trunk/SUAVE/Methods/Propulsion/propeller_design.py

```diff
@@ -118,37 +118,15 @@ def propeller_design(prop,number_of_stations=20):
     if  a_pol != None and a_loc != None:
         if len(a_loc) != N:
             raise AssertionError('\nDimension of airfoil sections must be equal to number of stations on propeller')
-    
+        airfoil_flag = True  
     else:
-        # Import Airfoil from regression
-        print('\nNo airfoils specified for propeller or rotor airfoil specified. \nDefaulting to NACA 4412 airfoils that will provide conservative estimates.') 
-        import os
-        ospath    = os.path.abspath(__file__)
-        separator = os.path.sep
-        path      = ospath.replace('\\','/').split('trunk/SUAVE/Methods/Propulsion/propeller_design.py')[0] \
-            + 'regression' + separator + 'scripts' + separator + 'Vehicles' + separator 
-        a_geo  = [ path +  'NACA_4412.txt'] 
-        a_pol  = [[path +  'NACA_4412_polar_Re_50000.txt' ,
-                   path +  'NACA_4412_polar_Re_100000.txt' ,
-                   path +  'NACA_4412_polar_Re_200000.txt' ,
-                   path +  'NACA_4412_polar_Re_500000.txt' ,
-                   path +  'NACA_4412_polar_Re_1000000.txt' ]]   # airfoil polars for at different reynolds numbers  
-       
-        # 0 represents the first airfoil, 1 represents the second airfoil etc. 
-        a_loc = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]  
-    
-        prop.airfoil_geometry        = a_geo
-        prop.airfoil_polars          = a_pol     
-        prop.airfoil_polar_stations  = a_loc     
+        airfoil_flag    = False   
```

Comment:
It may be worth putting an a print here too like before. Defaulting to a scaled DAE51.

Reply:
Comment placed